feature-group-definitions/check-visibility.yml

```
@@ -1,4 +1,5 @@
 name: checkVisibility()
+description: The `Element.checkVisibility()` method checks if an element is visible.
```

I wasn't sure how this compared to something like intersection observer. My attempt to clarify:

```suggestion
description: The `Element.checkVisibility()` method checks if an element is visible. For example, it checks whether an element has the style `display: none`.
```